static/app/views/dashboardsV2/addWidget.tsx

```diff
@@ -56,7 +56,6 @@ function AddWidget({onAddWidget, onOpenWidgetBuilder, orgFeatures}: Props) {
       <InnerWrapper onClick={onClick}>
         <AddButton
           data-test-id="widget-add"
-          onClick={onClick}
```

Comment:
I think this is safe to remove since we have an onClick on the InnerWrapper component. Testing this locally looked good.